From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Mon, 3 Apr 2017 15:34:03 +0100
Subject: [PATCH] data: Use stateless system directories for d-bus, PAM and
 default rules

[2021-06-03]: In the 0.119 release, the D-Bus policy file is now read from the
              /usr/share location, so we dropped that part of the patch.

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 data/Makefile.am              | 2 +-
 src/polkitbackend/Makefile.am | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index e805260..c29f296 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -25,7 +25,7 @@ $(dbusconf_DATA): $(dbusconf_in_files) Makefile
 # ----------------------------------------------------------------------------------------------------
 
 if POLKIT_AUTHFW_PAM
-pamdir = $(sysconfdir)/pam.d
+pamdir = $(datadir)/pam.d
 pam_DATA = polkit-1
 endif
 
diff --git a/src/polkitbackend/Makefile.am b/src/polkitbackend/Makefile.am
index 7e3c080..a1295da 100644
--- a/src/polkitbackend/Makefile.am
+++ b/src/polkitbackend/Makefile.am
@@ -64,7 +64,7 @@ libpolkit_backend_1_la_LIBADD =                               		\
 	$(LIBJS_LIBS)							\
         $(NULL)
 
-rulesdir = $(sysconfdir)/polkit-1/rules.d
+rulesdir = $(datadir)/polkit-1/rules.d
 rules_DATA = 50-default.rules
 
 # ----------------------------------------------------------------------------------------------------
